tests/__mocks__/lightning/actions.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
@sam-knox we also introduce a new quick action to launch the refund from the Payment record detail page.
<img width="1240" alt="Screen Shot 2022-05-20 at 1 52 08 PM" src="https://user-images.githubusercontent.com/57271120/169609012-6ee5ddb5-42e0-427f-800d-a7ba3811e484.png">
.